cloudera-integration/csd/descriptor/service.sdl

```diff
@@ -103,6 +103,81 @@
         ]
       }
     },
+    {
+      "name": "LOG_COLLECTOR_KAFKA",
+      "label": "Log Collector Kafka",
+      "pluralLabel": "Log Collectors Kafka",
+      "inExpressWizard": true,
+      "jvmBased" : "true",
+      "parameters": [
+        {
+          "name": "topic",
+          "label": "Kafka topic",
+          "description": "Log collector Kafka Topic",
+          "required": "false",
+          "type": "string",
+          "default": "pulse-kafka-topic"
+        },
+        {
+          "name": "log.collector.max.heap.size",
+          "label": "Log collector max java heap size",
+          "description": "Maximum size for the Java process heap memory. Passed to Java -Xmx. Measured in megabytes.",
+          "type": "memory",
+          "default": 4096,
+          "min": 50,
+          "softMin": 256,
+          "softMax": 16384,
+          "unit": "megabytes",
```

Comment:
I think the unit needs to be in bytes for the JVM. If you try to use MB the CSD doesn't automatically convert the format for the JVM. Make sure you take your example off of the master branch, I tried doing the same thing earlier and it doesn't work.

Reply:
applied this for Log Collector, Log Collector Kafka, Collection Roller, and Alert Engine roles